### PR TITLE
Add manuscript links to readme

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Propose moving the table from #731 to the front page, to help orient people

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,18 @@
...
+| Nutraceuticals | Dietary Supplements and Nutraceuticals Under Investigation for COVID-19 Prevention and Treatment  | ✔️ | [Published](https://doi.org/10.1128/mSystems.00122-21) 🎉 | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/nutraceuticals-v2); [Preprint](http://arxiv.org/abs/2102.02250) |
+| Methods (Cyberinfrastructure) | An Open-Publishing Response to the COVID-19 Infodemic | ✔️ | Accepted | [DISCO 2021](https://infoqualitylab.org/events/disco2021/) | [Release](https://github.com/greenelab/covid19-review/releases/tag/methods-v2); [Preprint](https://arxiv.org/abs/2109.08633)
+| Pharmaceuticals | Identification and Development of Therapeutics for COVID-19 | ✔️ | Accepted | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/pharmaceuticals-v3); [Preprint](http://arxiv.org/abs/2103.02723)
+| Pathogenesis | Pathogenesis, Symptomatology, and Transmission of SARS-CoV-2 through analysis of Viral Genomics and Structure | #744 | Ready for resubmission | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/pathogenesis-v2); [Preprint](http://arxiv.org/abs/2102.01521)
```
Comment (agitter) [inline README.md:17]:
```suggestion
| [Pathogenesis](https://greenelab.github.io/covid19-review/#pathogenesis-symptomatology-and-transmission-of-sars-cov-2-through-analysis-of-viral-genomics-and-structure) | Pathogenesis, Symptomatology, and Transmission of SARS-CoV-2 through analysis of Viral Genomics and Structure | #744 | Ready for resubmission | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/pathogenesis-v2); [Preprint](http://arxiv.org/abs/2102.01521)
```
What do you think about linking the first column to where the paper appears in the merged review?  That will help readers find the absolute latest version if they're interested.  Those links should be stable as long as the section titles do not change.

We can get the URLs from the table of contents in the HTML manuscript.

**Review by agitter — CHANGES_REQUESTED** [review]
This looks great.  The original suggestion to add a splash page came from [this reader](https://twitter.com/BioinfoTools/status/1440061438429519881).

Should we order the table rows as they appear in the merged review?  Sorting by submission date made sense when the table was oriented toward authors, but now it is for external readers as well.

File: README.md [hunk 2]
```diff
@@ -8,6 +8,18 @@
 [![Gitter](https://badges.gitter.im/covid19-review/community.svg)](https://gitter.im/covid19-review/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 <!-- usage note: delete CI badges above for services not used by your manuscript -->
 
+Project Status:
+| Section | Title | Status Issue | Submission Status | Venue | Links
+| -- | -- | -- | -- |-- | -- |
+| Nutraceuticals | Dietary Supplements and Nutraceuticals Under Investigation for COVID-19 Prevention and Treatment  | ✔️ | [Published](https://doi.org/10.1128/mSystems.00122-21) 🎉 | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/nutraceuticals-v2); [Preprint](http://arxiv.org/abs/2102.02250) |
+| Methods (Cyberinfrastructure) | An Open-Publishing Response to the COVID-19 Infodemic | ✔️ | Accepted | [DISCO 2021](https://infoqualitylab.org/events/disco2021/) | [Release](https://github.com/greenelab/covid19-review/releases/tag/methods-v2); [Preprint](https://arxiv.org/abs/2109.08633)
```
Comment (agitter) [inline README.md:15]:
```suggestion
| [Methods (Cyberinfrastructure)](https://greenelab.github.io/covid19-review/#an-open-publishing-response-to-the-covid-19-infodemic) | An Open-Publishing Response to the COVID-19 Infodemic | ✔️ | Accepted | [DISCO 2021](https://infoqualitylab.org/events/disco2021/) | [Release](https://github.com/greenelab/covid19-review/releases/tag/methods-v3); [Preprint](https://arxiv.org/abs/2109.08633)
```

Updating with the latest release I just made.

File: README.md [hunk 3]
```diff
@@ -8,6 +8,18 @@
 [![Gitter](https://badges.gitter.im/covid19-review/community.svg)](https://gitter.im/covid19-review/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 <!-- usage note: delete CI badges above for services not used by your manuscript -->
 
+Project Status:
+| Section | Title | Status Issue | Submission Status | Venue | Links
+| -- | -- | -- | -- |-- | -- |
+| [Pathogenesis](https://greenelab.github.io/covid19-review/#pathogenesis-symptomatology-and-transmission-of-sars-cov-2-through-analysis-of-viral-genomics-and-structure) | Pathogenesis, Symptomatology, and Transmission of SARS-CoV-2 through analysis of Viral Genomics and Structure | #744 | Ready for resubmission | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/pathogenesis-v2); [Preprint](http://arxiv.org/abs/2102.01521)
```
Comment (agitter) [inline README.md:14]:
```suggestion
| [Pathogenesis](https://greenelab.github.io/covid19-review/#pathogenesis-symptomatology-and-transmission-of-sars-cov-2-through-analysis-of-viral-genomics-and-structure) | Pathogenesis, Symptomatology, and Transmission of SARS-CoV-2 through analysis of Viral Genomics and Structure | ✔️ | Accepted | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/pathogenesis-v2); [Preprint](http://arxiv.org/abs/2102.01521)
```

**Review by agitter — APPROVED** [review]

**Comment by rando2** [comment]
Thank you for making that update @agitter! I got notice today they posted the new version of the preprint, so I believe everything should be up to date now!

**Pull request merged**